README.md

```diff
@@ -196,13 +196,18 @@ Usage:
 ## Windows nodes Kubernetes component upgrade
 
 When a new version of WMCO is released that is compatible with the current cluster version, an operator upgrade will 
-take place which will result in the Kubernetes components in the Windows Machine to be upgraded. For a non-disruptive 
-upgrade, WMCO terminates the Windows Machines configured by previous version of WMCO and recreates them using the
-current version. This is done by deleting the Machine object that results in the drain and deletion of the Windows node.
+take place which will result in the Kubernetes components in the Windows Machine to be upgraded. For a non-disruptive
+upgrade, WMCO processes the Windows Machine that was configured by the previous version of the operator with the
+following steps:
+- drain and cordon the associated Node object
+- re-configure it using the current version
```

Comment:
How about changing current version-> new version? for clarity

Reply:
addressed in  6c2e21f